MicrosoftFluentUI.podspec

```diff
@@ -1,151 +1,64 @@
-  s.source       = { :git => "https://github.com/microsoft/fluentui-apple.git", :tag => "#{s.version}" }
-  s.swift_version = "5.0"
+  s.source       = { :git => "https://github.com/microsoft/fluentui-apple.git", :branch => 'vnext-prototype', :tag => "#{s.version}" }
+  s.swift_version = "5.3.1"
   s.module_name = 'FluentUI'
 
 
 # iOS
 
   s.ios.deployment_target = "13.0"
 
-  s.subspec 'Calendar_ios' do |calendar_ios|
-    calendar_ios.platform = :ios
-    calendar_ios.dependency 'MicrosoftFluentUI/Presenters_ios'
-    calendar_ios.source_files = ["ios/FluentUI/Calendar/**/*.{swift,h}",
-                                 "ios/FluentUI/Date Time Pickers/**/*.{swift,h}"]
-  end
-
-  s.subspec 'Card_ios' do |card_ios|
-    card_ios.platform = :ios
-    card_ios.dependency 'MicrosoftFluentUI/Controls_ios'
-    card_ios.source_files = ["ios/FluentUI/Card/**/*.{swift,h}"]
-  end
-
-  s.subspec 'Controls_ios' do |controls_ios|
-    controls_ios.platform = :ios
-    controls_ios.dependency 'MicrosoftFluentUI/Core_ios'
-    controls_ios.source_files = ["ios/FluentUI/Badge Field/**/*.{swift,h}",
-                                 "ios/FluentUI/Controls/**/*.{swift,h}",
-                                 "ios/FluentUI/Navigation/**/*.{swift,h}",
-                                 "ios/FluentUI/People Picker/**/*.{swift,h}",
-                                 "ios/FluentUI/Table View/**/*.{swift,h}"]
-  end
-
   s.subspec 'Core_ios' do |core_ios|
     core_ios.platform = :ios
     core_ios.resource_bundle = { 'FluentUIResources-ios' => ["apple/Resources/**/*.{json,xcassets}",
                                                              "ios/FluentUI/**/*.{storyboard,xib,xcassets,strings,stringsdict}"] }
-    core_ios.source_files = ["ios/FluentUI/Configuration/**/*.{swift,h}",
-                             "ios/FluentUI/Core/**/*.{swift,h}",
-                             "ios/FluentUI/Extensions/**/*.{swift,h}"]
+    core_ios.source_files = ["ios/FluentUI/Core/Colors.swift",
+                             "ios/FluentUI/Core/FluentUIFramework.swift",
+                             "ios/FluentUI/Core/Fonts.swift",
+                             "ios/FluentUI/Core/iOS13_4_compatibility.swift",
+                             "ios/FluentUI/Core/UIViewAdapter.swift",
+                             "ios/FluentUI/Extensions/Calendar+Extensions.swift",
+                             "ios/FluentUI/Extensions/CharacterSet+Extension.swift",
+                             "ios/FluentUI/Extensions/String+Extension.swift",
+                             "ios/FluentUI/Extensions/UIColor+Extensions.swift",
+                             "ios/FluentUI/Extensions/UIFont+Extension.swift",
+                             "ios/FluentUI/Extensions/UIImage+Extensions.swift",
+                             "ios/FluentUI/Extensions/UIScreen+Extension.swift",
+                             "ios/FluentUI/Design Token System/**/*.{swift,h}"]
+  end
+
+  s.subspec 'Avatar_ios' do |avatar_ios|
```

Comment:
Would it make sense to have a subspec for vnext only instead?

Reply:
We're basically setting the stage to allow consumers of FluentUIVnext to get individual controls if they want to. If they don't define the subspec to be consumed, they'll just get it all.